web/templates/keyboard_shortcuts.hbs

```diff
@@ -22,19 +22,19 @@
                 </tr>
                 <tr>
                     <td class="definition">{{t 'Cancel compose and save draft' }}</td>
-                    <td><span class="hotkey"><kbd>Esc</kbd> or <kbd>Ctrl</kbd> + <kbd>[</kbd></span></td>
+                    <td><span class="hotkey"><kbd>Esc</kbd> {{t 'or' }} <kbd>Ctrl</kbd> + <kbd>[</kbd></span></td>
                 </tr>
                 <tr>
                     <td class="definition">{{t 'View drafts' }}</td>
                     <td><span class="hotkey"><kbd>D</kbd></span></td>
                 </tr>
                 <tr>
                     <td class="definition">{{t 'Next message' }}</td>
-                    <td><span class="hotkey"><kbd class="arrow-key">↓</kbd> or <kbd>J</kbd></span></td>
+                    <td><span class="hotkey"><kbd class="arrow-key">↓</kbd> {{t 'or' }} <kbd>J</kbd></span></td>
                 </tr>
                 <tr>
                     <td class="definition">{{t 'Last message' }}</td>
-                    <td><span class="hotkey"><kbd>End</kbd> or <kbd>Shift</kbd> + <kbd>G</kbd></span></td>
+                    <td><span class="hotkey"><kbd>End</kbd> {{t 'or' }} <kbd>Shift</kbd> + <kbd>G</kbd></span></td>
```

Comment:
Can you post in `#translating` about this one? Generally you can't translated individual words like this -- you need to translate sentences because things like word order vary -- but it seems like maybe this could be correct in general?

Reply:
Posted here https://chat.zulip.org/#narrow/stream/58-translation/topic/conjunction.20translation

Reply:
This can probably be done properly using `Intl.ListFormat`, using the non-conjunction style.